src/node_api.cc

```diff
@@ -1618,6 +1637,30 @@ napi_status napi_get_value_string_length(napi_env e,
   return GET_RETURN_STATUS();
 }
 
+// Copies a JavaScript string into a LATIN-1 string buffer. The result is the
+// number
+// of bytes copied into buf, including the null terminator. If the buf size is
+// insufficient, the string will be truncated, including a null terminator.
+napi_status napi_get_value_string_latin1(napi_env e,
+                                         napi_value value,
+                                         char* buf,
+                                         int bufsize,
+                                         int* result) {
+  NAPI_PREAMBLE(e);
+
+  v8::Local<v8::Value> val = v8impl::V8LocalValueFromJsValue(value);
+  RETURN_STATUS_IF_FALSE(val->IsString(), napi_string_expected);
+
+  int copied = val.As<v8::String>()->WriteOneByte(
```

Comment:
If `buf` is null, just return the length. Refer to the updated implementation of the other `napi_get_value_string_` APIs, after rebasing.